pyrogram/types/bots_and_keyboards/keyboard_button_poll_type.py

```diff
@@ -0,0 +1,34 @@
+#  Pyrogram - Telegram MTProto API Client Library for Python
+#  Copyright (C) 2017-2021 Dan <https://github.com/delivrance>
+#
+#  This file is part of Pyrogram.
+#
+#  Pyrogram is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Lesser General Public License as published
+#  by the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  Pyrogram is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with Pyrogram.  If not, see <http://www.gnu.org/licenses/>.
+
+
+class KeyboardButtonPollType:
+    """This object represents type of a poll,
+    which is allowed to be created and sent when the corresponding button is pressed.
+    
+    Parameters:
+        type (``str``):
+            If quiz is passed, the user will be allowed to create only polls in the quiz mode.
+            If regular is passed, only regular polls will be allowed.
+            Otherwise, the user will be allowed to create a poll of any type.
+    """
+    def __init__(
+        self,
+        type: str
```

Comment:
It seems like this can be optional, https://core.telegram.org/bots/api/#keyboardbuttonpolltype.

Reply:
I couldn't get the "regular" to work. (I don't know which raw type to use)
only quiz=True seems to work.